pages/linux/debtap.md

```diff
@@ -0,0 +1,24 @@
+# debtap
+
+> Convert .deb packages into Arch Linux packages.
+> More information: <https://github.com/helixarch/debtap>.
+
+- Update debtap database (before first run):
+
+`sudo debtap -u`
+
+- Convert a package:
+
+`debtap {{package.deb}}`
+
+- Convert a package (bypass all questions, not recommended):
+
+`debtap -Q {{package.deb}}`
+
+- Then install with pacman:
+
+`sudo pacman -U {{package.pkg.tar.zst}}`
```

Comment:
We don't use to add examples to other commands, that's why we can add the `See also` note. Please remove this example.